apps/rehabstod/src/pages/CurrentSickLeaves/sickLeaveSlice.ts

```diff
@@ -1,15 +1,19 @@
 /* eslint-disable no-param-reassign */
 import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import _ from 'lodash'
```

Comment:
We could import specific `lodash` functions for tree shaking to work efficiently with `import isEqual from 'lodash/isEqual'`